src/Microsoft.Azure.SignalR.Management/HubInstanceFactories/ServiceHubLifetimeManagerFactory.cs

```diff
@@ -34,9 +34,9 @@ public IServiceHubLifetimeManager Create(string hubName)
                     }
                 case ServiceTransportType.Transient:
                     {
-                        return new RestHubLifetimeManager(hubName, _context.ServiceEndpoints.Single(), _context.ProductInfo, _context.ApplicationName);
+                        return new RestHubLifetimeManager(hubName, new ServiceEndpoint(_options.ConnectionString), _options.ProductInfo, _options.ApplicationName);
```

Comment:
>ServiceEndpoint [](start = 71, length = 15)

what‘s the difference between new ServiceEndpoint(_options.ConnectionString) and _context.ServiceEndpoints.Single()

Reply:
We don’t use `ContextOptions` now, so `_context.ServiceEndpoints.Single()` won’t contains value. What’s more, `ServiceManagerOptions.ServiceEndpoints` won’t contain value of `ServiceManagerOptions.ConnectionString`.

Reply:
I see. ContextOptions doesn't have connection string